src/Orchard.Cms.Web/Modules/Orchard.Commons/Orchard.Commons.csproj

```diff
@@ -1,16 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
+  <Import Project="..\..\..\..\build\common.props" />
```

Comment:
I am not complaining about it, but just mentioning there is a way to inject this file automatically without requiring the `<Import>` tag in all projects. http://blog.seravy.com/directory-build-targets-solution-wide-msbuild-target-part-2/